3/debian-10/rootfs/opt/bitnami/scripts/moodle-env.sh

```diff
@@ -82,6 +84,8 @@ export MOODLE_INSTALL_EXTRA_ARGS="${MOODLE_INSTALL_EXTRA_ARGS:-}" # only used du
 export MOODLE_SITE_NAME="${MOODLE_SITE_NAME:-New Site}" # only used during the first initialization
 export MOODLE_HOST="${MOODLE_HOST:-}" # only used during the first initialization
 export MOODLE_CRON_MINUTES="${MOODLE_CRON_MINUTES:-1}"
+export MOODLE_REVERSEPROXY="${MOODLE_REVERSEPROXY:-}" # only used during the first initialization
+export MOODLE_SSLPROXY="${MOODLE_SSLPROXY:-}" # only used during the first initialization
```

Comment:
Default value should be "no":
```suggestion
export MOODLE_REVERSEPROXY="${MOODLE_REVERSEPROXY:-no}" # only used during the first initialization
export MOODLE_SSLPROXY="${MOODLE_SSLPROXY:-no}" # only used during the first initialization
```